src/CUSBCommunication.cpp

```diff
@@ -33,6 +33,8 @@ CUSBCommunication::CUSBCommunication(string device) : transfer(NULL), isoReceive
 	libusb_device **deviceList;
 	int busNr = 0;
 	int deviceNr = 0;
+	int currentConfig = -1;
+	const int desiredConfig = 1;
```

Comment:
I think it would be better to have `desiredConfig` as a macro in CUSBCommunication.h, as it is a "magic constant".

Reply:
What would be a fitting name for that macro?
Would you agree with one of the following, if yes, which one do you prefer?
```
#define	USB_DEVICE_DEFAULT_CONFIGURATION (1)
#define	USB_DEVICE_DESIRED_CONFIGURATION (1)
#define	USB_DEVICE_TARGET_CONFIGURATION (1)
```

Reply:
I would go with USB_DEVICE_TARGET_CONFIGURATION

Reply:
Your suggestion to use a macro has now been implemented.